### PR TITLE
Improved comments in default.conf

### PR DESCRIPTION
**Pull request by jsmeix**
* Type: **Enhancement**

* Impact: **Low**

* Reference to related issue (URL):
https://github.com/rear/rear/issues/2870#issuecomment-1261045942

* Brief description of the changes in this pull request:

In default.conf better description
of the BACKUP=EXTERNAL config variables, see
https://github.com/rear/rear/issues/2870#issuecomment-1261045942
and some general improvements of the comments
e.g. removed the colloquial word 'stuff' where not useful
and in particular make blocks of config variables
that belong together more clear by using
```
####
.
.
.
####
```
as indicators where a block begins and ends.

### CODE REVIEW

**Review by pcahyna — APPROVED** [review]

**Comment by jsmeix** [comment]
I had a closer look what is actually implemented which results my latest
https://github.com/rear/rear/pull/2872/commits/7714104b6653185811e2faa75f5f2004e9b80e01

**Comment by jsmeix** [comment]
@rear/contributors 
if there are no objections I would like to merge it
tomorrow afternoon.

File: usr/share/rear/conf/default.conf [hunk 1]
```diff
@@ -2887,38 +2887,46 @@ RECREATE_USERS_GROUPS="no"
 ####
 # BACKUP=EXTERNAL
 ##
-# Custom command backup stuff.
+# Custom command backup stetup.
```
Comment (pcahyna) [inline usr/share/rear/conf/default.conf:2890]: typo

Comment (jsmeix) [reply usr/share/rear/conf/default.conf]: @pcahyna 
thank you for your careful proofreading!

Fixed via
https://github.com/rear/rear/pull/2872/commits/2b1533bc5b21ff0da82b82631b9c6b83408bcb45

**Pull request merged**